generators/init-product/templates/plain/build.js

```diff
@@ -606,8 +606,26 @@ function strObject(items) {
   return obj;
 }
 
+/**
+ * Use Dockerfiles from the product repo if present. Otherwise use default Dockerfiles.
+ * @param {object} p A phovea_product.json entity i.e., web, server.
+ * @param {string} cwd Directory in which to run docker command.
+ */
+function getDockerfilePath(p, cwd) {
+  const customDockerFilePath = `docker/${p.label}/Dockerfile`;
+  const hasCustomDockerfile = fs.existsSync(customDockerFilePath);
+
+  if (hasCustomDockerfile) {
+    return path.join(__dirname, `${customDockerFilePath}`);
+  }
+
+  return `${cwd} /'deploy/Dockerfile'`;
```

Comment:
@oltionchampari I think you have to remove the empty space

Reply:
You are right, thanks for the catch!